### DISCUSSION TITLE
Can pywa be used to send messages to a Channel?

### DISCUSSION THREAD

**Discussion by cseguinlz** [david-lev/pywa#43] [Q&A] [answered]
Hi,
looking at this note in the docs:

> The to parameter must be a phone number with the country code. For example: +972123456789, 16315551234.

Just wondering would the `to` param accept a Channel id, "120363217653060227@newsletter", as a recipient? Would that work?

**Comment by yehuda-lev** [comment]
No. You cannot send a message in groups and channels - because you are not a member of them. If you manage to connect your bot to a group or channel, you may be able to send messages in them.. (try it 😆)

**Reply by cseguinlz** [reply]
ahh sorry, I meant, mine own channel, use the API to programmatically broadcast messages...

**Reply by yehuda-lev** [reply, marked as answer]
You can't 😔

**Reply by cseguinlz** [reply]
Ok, thanks for answer, you've saved me some time, much appreciated
